src/commands/addBinding/settingSteps/eventHub/EventHubListStep.ts

```diff
@@ -3,30 +3,53 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { EHNamespace, EventHubManagementClient } from '@azure/arm-eventhub';
+import { EventHubManagementClient, Eventhub } from '@azure/arm-eventhub';
 import { uiUtils } from '@microsoft/vscode-azext-azureutils';
-import { AzureWizardPromptStep } from '@microsoft/vscode-azext-utils';
+import { AzureWizardPromptStep, IAzureQuickPickItem, IWizardOptions, nonNullValueAndProp } from '@microsoft/vscode-azext-utils';
 import { localize } from '../../../../localize';
-import { promptForResource } from '../../../../utils/azure';
 import { createEventHubClient } from '../../../../utils/azureClients';
 import { nonNullProp } from '../../../../utils/nonNull';
+import { EventHubCreateStep } from './EventHubCreateStep';
 import { IEventHubWizardContext } from './IEventHubWizardContext';
 
 export class EventHubListStep extends AzureWizardPromptStep<IEventHubWizardContext> {
     public async prompt(context: IEventHubWizardContext): Promise<void> {
-        const namespaceName: string = nonNullProp(context, 'namespaceName');
-        const resourceGroupName: string = nonNullProp(context, 'resourceGroupName');
+        if (!context.eventHubsNamespace) {
+            // if there is no event hub namespace, then we can't create an event hub but
+            // getSubWizard only gets called if prompt does so we can't use shouldPrompt to stiffle this prompt
```

Comment:
Nit
```suggestion
            // getSubWizard only gets called if prompt does so we can't use shouldPrompt to stifle this prompt
```

Reply:
stiffle reminds me of puffles from club penguin <img src="https://media3.giphy.com/media/70PXZeazUazK/giphy.gif"/>